config/database.yml

```diff
@@ -6,16 +6,14 @@
 # PostgreSQL. Versions 8.2 and up are supported.
 #
 base: &generic
-  adapter: <%= ENV['RAILS_DB_ADAPTER'] || 'sqlite3' %>
+  adapter: sqlite3
```

Comment:
Why make `sqlite3` the default here.
You want the opposite.
Either make PostgreSQL the default or leave it as it was and set the environment variable `RAILS_DB_ADAPTER`.